engine/table/src/test/java/io/deephaven/engine/table/impl/QueryTableCrossJoinTestBase.java

```diff
@@ -409,10 +409,8 @@ private void testStaticJoin(final String[] types, final int[] cardinality, int m
             TableTools.showWithRowSet(chunkedCrossJoin, 100);
         }
 
-        QueryTable.USE_CHUNKED_CROSS_JOIN = false;
         final Table nonChunkedCrossJoin =
                 left.join(right, List.of(JoinMatch.parse("sharedKey")), emptyList(), numRightBitsToReserve);
-        QueryTable.USE_CHUNKED_CROSS_JOIN = true;
```

Comment:
This is no longer valid.  You can simulate it without all the fidelity/performance nuggets/iteration of column names, etc. that used to exist since you know exactly the columns you want.